content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -8,8 +8,15 @@ weight: 10
 
 This page will show you how to get started with OpenTelemetry in Node.js.
 
-You will learn how you can instrument a simple application automatically, in
-such a way that [traces][], [metrics][] and [logs][] are emitted to the console.
+You will learn how to automatically instrument both [traces][] and [metrics][] and log them to the console.
+ [Automatically](https://opentelemetry.io/docs/concepts/instrumentation/automatic/) 
+instrumenting means that the application is made observable without 
+touching the application code.
```

Comment:
```suggestion
You will learn how to instrument both [traces][] and [metrics][] and log them to the console.
```

See @pellared's comment:  https://github.com/open-telemetry/opentelemetry.io/pull/3302/files#r1336747605

Reply:
Im relatively new to pull request...I had agreed with the changes from @pellared where it included the word "automatic" and a link to the automatic page.

Reply:
@pellared suggested to drop the word "automatic". There was a related change incorporated in the linked PR, which got removed to avoid a conflict with your PR: https://github.com/open-telemetry/opentelemetry.io/pull/3308/commits/589e51b37f0b2518cb33b47eddb1a5191e07c31c

Reply:
Oh yeah..I remember..lemmi correct that